concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -153,7 +153,7 @@ local base_buildpackagejob = {
       },
     },
```

Comment:
I'm skipping this changes as I know you are still working on those.

Reply:
It turns out the diff view is messed up. Nothing to change. Only changes from me are lines 684 and 707 in this file.

Reply:
ok